github.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/google/go-github/github"
 )
 
-var _ = github.ScopeAdminOrg
+var X = fmt.Sprintf("%s")
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var X should have comment or be unexported